src/Features/Core/Diagnostics/DiagnosticAnalyzerService.cs

```diff
@@ -15,29 +17,35 @@ namespace Microsoft.CodeAnalysis.Diagnostics
     [Shared]
     internal partial class DiagnosticAnalyzerService : IDiagnosticAnalyzerService
     {
-        private readonly WorkspaceAnalyzerManager _workspaceAnalyzerManager;
+        private readonly HostAnalyzerManager _hostAnalyzerManager;
```

Comment:
Thanks for the rename, I have always felt WorkspaceAnalyzerManager was the wrong term I chose, given that it is independent of workspace.